### PR TITLE
Api role

### PR DESCRIPTION
**Pull request by brainstorm**
Except the port forwarding issue, things seem to click together :)

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -41,7 +41,7 @@ arvados_cluster = {
         :ipaddress => "10.10.10.8",
         :memory => 512,
         :ansible_role => "shell"
-    }
+    },
```
Comment (guillermo-carrasco) [inline Vagrantfile]: this comma will break the recipe, I think, eller?

File: cluster-vm [hunk 2]
```diff
@@ -20,3 +22,6 @@
 
 [sso]
 10.10.10.8
+
+[api]
+10.10.10.9
```
Comment (guillermo-carrasco) [inline cluster-vm]: could you remove this as well please? :)

File: roles/arvados/tasks/main.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 # so unfortunately for RPM distributions like CentOS this needs to be done manually
 - name: Register the Curoverse signing key in RPM’s database
   shell: >
-    gpg --keyserver pool.sks-keyservers.net --recv-keys 1078ECD7 &&
+    gpg --keyserver pgp.mit.edu --recv-keys 1078ECD7 &&
```
Comment (guillermo-carrasco) [inline roles/arvados/tasks/main.yaml:15]: why this change? Is not that I care if it works, but since it was working before.

Comment (brainstorm) [inline roles/arvados/tasks/main.yaml:15]: The pgp pool contains plenty of unreliable pgp hosts that timeout. During my testing it was failing like 70% of the time. pgp.mit.edu might not be the best, but it has been holding up for a long time for me.

File: Vagrantfile (hunk 1 above)
Comment (brainstorm) [inline Vagrantfile]: Nope, it seems to behave as Perl where the last comma is optional... I'll remove it anyway.

File: roles/arvados/tasks/main.yaml (hunk 3 above)
Comment (guillermo-carrasco) [inline roles/arvados/tasks/main.yaml:15]: awesome, thanks!

**Comment by guillermo-carrasco** [comment]
Thank you @brainstorm !
